Eastern-Research-Group/csb-rebate-forms-app
# Feature/add initial close out form

## Related Issues:
* CSBAPP-5

## Main Changes:
Adds a new stub route and component for the Close-Out form. As there's lots of duplicated code between the Application form page and the Payment Request form page, instead of further duplicating it in the new Close-Out form page, I took the opportunity to refactor a good bit of that code, which should make the app more maintainable in the long run.

## Steps To Test:
1. Run app – all should still function as normal.
2. A new route exists at `/close-out/:id` (NOTE: just a placeholder page for now)

## TODO:
- [ ] Update the server app routes to support fetching the close-out data, and flesh out the the Close-Out form route component to display the close-out form component
